File: selection_sort.py
k=[]
n=int(input('enter range'))
for i in range(n):
    k.append(int(input('enter')))
print("before swapping",k)
for i in range(n-1):
    min=i
    
    for j in range(i+1,n):
        if(k[j]<k[min]):
            min=j
    temp=k[min]
    k[min]=k[i]
    k[i]=temp
print("after sorting",k)        

    

File: selection_sort.c
#include <stdio.h>
void main() {
    int i,arr[10],j,min,m,n=6,k,temp;
    for ( i = 0; i < n; i++)
    {
        printf("enter");
        scanf("%d",&arr[i]);
    }
    for ( j= 0; j <n-1; j++)
    {
        min=j;
        for ( k =j+1; k <n ; k++)
        {
            if(arr[k]<arr[min])
            {
                temp=arr[min];
                arr[min]=arr[k];
                arr[k]=temp;
            }
        }
        
    }
    
    for ( m = 0; m< n; m++)
    {
        printf("%d",arr[m]);
    }
    


}